### ISSUE TITLE
Pull Request Trigger type support for Azure Devops

### ISSUE THREAD

**Issue by alaeddineG** [bitrise-io/bitrise-webhooks#80]
The Code Push webhook on Vsts (Azure Devops) work great with bitrise. But is there any way (a workaround) to be able to do a Pull Request rigger.
I mean I dont want to have every code push to any branch in our AzureDevops repo to trigger a build on bitrise. 

How to do that ?

**Comment by fehersanyi-bitrise** [comment]
Hello there! you can configure your triggers on the wokrflow editors triggers tab three way.
setting up for push, you can name a branch and only that will trigger the build.
you can set up for pull requests, with a from - to setting
and you can set up triggers for git tags as well, so only specific commits will trigger a build. like a "ci" tag on the commit will trigger a bitrise build

**Comment by alaeddineG** [comment]
The Code push trigger work perfectly with the proper naming pattern. I've set a trigger for any ode push on `feature/*` and for any push to that kind of branch there is build triggered.

What is not working/supported is when a pul request is created/updated :
starting from no trigger at all in an app in Bitrise and setup a Pull request trigger in the App (from any branch to develop ) creating a pull request in AzureDevops wont trigger any build because the webhook in Bitrise does not support PR Create/Update 

![screenshot 2018-11-26 17 11 33](https://user-images.githubusercontent.com/2363851/49045383-d4ac7980-f19e-11e8-948c-2183f3d4aa28.png)

There is no way to set a pull request trigger (Create&Update) in AzureDevOps and if I put the bitrise Webhook URL in there on a PR Created/Update setup and trigger a test it will fail.

**Comment by fehersanyi-bitrise** [comment]
Hy there!
Unfortunately, there is no workaround for this yet, azure webhooks work like this and we can't do anything about it :'(

**Comment by viktorbenei** [comment]
Hi @alaeddineG 👋 

Indeed the VSTS implementation only supports code push. I believe at that time there was no PR event at all 🤔 

If you have some time you can find the existing implementation at https://github.com/bitrise-io/bitrise-webhooks/blob/master/service/hook/visualstudioteamservices/visualstudioteamservices.go - pull requests to add these kind of functionality are always welcome! 😉

**Comment by viktorbenei** [comment]
@alaeddineG definitely! To bump its priority, can you please create a feature request at https://discuss.bitrise.io/c/feature-request ? I checked and there's no FR for VSTS/AzureDevOps, as a template here's the one for Gogs: https://discuss.bitrise.io/t/pull-request-for-gogs-repositories/3519

**Comment by viktorbenei** [comment]
Thanks @alaeddineG ! ;)
